### ISSUE TITLE
post deployment executed before database is up

### ISSUE THREAD

**Issue by jorgemoralespou** [ivanthelad/postgres-postdeploy#1]
You are executing the post deployment hook and waiting for the database up. Instead you should add a readinessProbe that tests when the database is up, and that will mark the pod as running, then the post deployment will be executed and no need to wait. (I think there might be a bug preventing this to work properly in 3.1, but once solved, you should use this approach)

**Comment by ivanthelad** [comment]
yup. i know. it was quickly hacked together for a 'poc'. to prove the 
concept.
thanks for the review. appreciate it
